generate/templates/manual/src/functions/copy.cc

```diff
@@ -20,3 +20,15 @@ void git_transfer_progress_dup(git_transfer_progress **out, const git_transfer_p
   *out = (git_transfer_progress *)malloc(sizeof(git_transfer_progress));
   memcpy(*out, arg, sizeof(git_transfer_progress));
 }
+
+git_remote_head *git_remote_head_dup(const git_remote_head *src) {
+  git_remote_head *dest = (git_remote_head *)malloc(sizeof(git_remote_head));
+  dest->local = src->local;
+  git_oid_cpy(&dest->oid, &src->oid);
+  git_oid_cpy(&dest->loid, &src->loid);
+  dest->name = strdup(src->name);
```

Comment:
I think it's always supposed to be defined, but you're right, we should err on the side of safety :man_facepalming:.